### PR TITLE
Feature/7 Use classnames on all components

### PR DESCRIPTION
**Pull request by jscottsmith**
Address #7 

## Changes

- Use `classnames` lib where applicable
- Fix some warnings related to the updated `select` knob API
  - `'': 'No Value',` => `'No Value': '',`
- Updated a couple stories to make them look better

### CODE REVIEW

File: _stories/layout/LayoutContainer/index.js [hunk 1]
```diff
@@ -14,9 +15,22 @@ const component = () => (
         onClick={action('layout_container_clicked')}
         className={text('Extra classes')}>
         <Row>
-            <Column md={4}>Example</Column>
-            <Column md={4}>Example</Column>
-            <Column md={4}>Example</Column>
+            <Column md={4}>
+                <Card option="underlined" className="-p-a-2">
```
Comment (mnicole) [inline _stories/layout/LayoutContainer/index.js]: ooh these look nice.

File: _stories/layout/LayoutContainer/index.js [hunk 2]
```diff
@@ -14,9 +15,22 @@ const component = () => (
         onClick={action('layout_container_clicked')}
         className={text('Extra classes')}>
         <Row>
-            <Column md={4}>Example</Column>
-            <Column md={4}>Example</Column>
-            <Column md={4}>Example</Column>
+            <Column md={4}>
```
Comment (edrpls) [inline _stories/layout/LayoutContainer/index.js]: Question, do these accept `md="4"` as well?

**Comment by mnicole** [comment]
can you also remove the trimString helper since we don't need it anymore?

File: _stories/layout/LayoutContainer/index.js (hunk 2 above)
Comment (mnicole) [reply _stories/layout/LayoutContainer/index.js]: yes they do

File: components/atoms/LoadingDots.jsx [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
-import trimString from '../utils/trimString';
+import cx from 'classnames';
 
-const LoadingDots = ({ whiteDots = false, size, className, style }) => {
-    const baseClass = 'gds-loading__dot',
-        whiteDotsClass = whiteDots ? `${baseClass}--white` : '',
-        sizeClass = size ? `${baseClass}--${size}` : '';
-
-    const classNames = trimString(`${baseClass} ${whiteDotsClass} ${sizeClass}`);
+const LoadingDots = ({ whiteDots, size, className, style }) => {
```
Comment (mnicole) [inline components/atoms/LoadingDots.jsx]: oh yeah i noticed on this component, there's a padding that is applied on the story that should be removed

Comment (jscottsmith) [reply components/atoms/LoadingDots.jsx]: oh good catch. Actually I was applying `className` twice in LoadingDots.

**Comment by jscottsmith** [comment]
Updated to address comments @mnicole.

**Pull request merged**